core/src/main/java/io/questdb/cairo/MapBeDroppedTableToken.java

```diff
@@ -24,11 +24,11 @@
 
 package io.questdb.cairo;
 
-public class ReverseTableMapItem {
+public class MapBeDroppedTableToken {
```

Comment:
This naming suggests that the table has already dropped, but usually it's not the case. When I see this

```
irNameToTableTokenMap.put(dirName, MapBeDroppedTableToken.of(token));
```

I think that it marks table as dropped but in fact it does not.